react-components/src/lib.js

```diff
@@ -1,3 +1,10 @@
 import * as apps from './apps';
 
+import React from "react";
+import ReactDOM from "react-dom";
+
+// gwt-react needs React and ReactDOM on the global object
+window.React = React;
+window.ReactDOM = ReactDOM;
```

Comment:
It seems after some research, that you should be able to accomplish this via the webpack config file, but I didn't have any luck getting that to work.  This is what the gwt-react project is doing, so I figured I'll do the same for now.